### PR TITLE
add tcgverse mainnet

### PR DESCRIPTION
**Pull request by akira-19**
added tcgverse mainnet

see chainlist
https://chainlist.org/chain/2400

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1196,6 +1196,22 @@
...
   },
+  "2400": {
+    "key": "2400",
+    "name": "TCG Verse Mainnet",
+    "shortName": "TCGV",
+    "chainId": 2400,
+    "network": "mainnet",
+    "multicall": "0xceC65DEE0b5012F1b7321b2647681F997c7204FC",
+    "rpc": [
+      "https://rpc.tcgverse.xyz"
```
Comment (ChaituVR) [inline src/networks.json:1207]: Hi @akira-19 This node has very less payload limit, It should work when we pass atleast 500 addresses into a multicall

File: src/networks.json [hunk 2]
```diff
@@ -1196,6 +1196,22 @@
     "start": 94085,
     "logo": "ipfs://QmaQxfwpXYTomUd24PMx5tKjosupXcm99z1jL1XLq9LWBS"
   },
+  "2400": {
+    "key": "2400",
+    "name": "TCG Verse Mainnet",
+    "shortName": "TCGV",
+    "chainId": 2400,
+    "network": "mainnet",
+    "multicall": "0xceC65DEE0b5012F1b7321b2647681F997c7204FC",
```
Comment (ChaituVR) [inline src/networks.json:1205]: Contract code here https://explorer.tcgverse.xyz/address/0xceC65DEE0b5012F1b7321b2647681F997c7204FC/contracts should be verified and published.

File: src/networks.json [hunk 3]
```diff
@@ -1196,6 +1196,22 @@
...
+    "rpc": [
+      "https://rpc.tcgverse.xyz"
+    ],
+    "explorer": {
+      "url": "https://explorer.tcgverse.xyz"
+    },
+    "start": 54248,
```
Comment (ChaituVR) [inline src/networks.json:1212]:
```suggestion
    "start": 57500,
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (akira-19) [reply src/networks.json:1207]: @ChaituVR Thank you. I'm checking.

Comment (akira-19) [reply src/networks.json:1207]: @ChaituVR It has been fixed.

File: src/networks.json (hunk 2 above)
Comment (akira-19) [reply src/networks.json:1205]: @ChaituVR The contract is verified. Sorry, line breaks are shown weirdly, but the contract is correct.

File: src/networks.json (hunk 3 above)
Comment (ChaituVR) [reply src/networks.json:1212]: Can apply this change?

Comment (akira-19) [reply src/networks.json]: @ChaituVR Done.

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
@akira-19 Can update the base branch? Or allow changes from maintainers  https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

**Comment by akira-19** [comment]
@ChaituVR updated base branch.

**Pull request merged**